src/server/wsgi.py

```diff
@@ -33,6 +35,52 @@ def new_wrapper() -> FlaskApp:
 """
 
 application = quantumleap.app
+
+def use_mqtt() -> bool:
+    env_var = BoolVar('USE_MQTT', False)
+    print(EnvReader().safe_read(env_var))
+    return EnvReader().safe_read(env_var)
+
+if use_mqtt():
+    application.config['MQTT_BROKER_URL'] = server.MQTT_HOST
+    application.config['MQTT_BROKER_PORT'] = server.MQTT_PORT
+    application.config['MQTT_USERNAME'] = server.MQTT_USERNAME
+    application.config['MQTT_PASSWORD'] = server.MQTT_PASSWORD
+    application.config['MQTT_KEEPALIVE'] = server.MQTT_KEEPALIVE
+    application.config['MQTT_TLS_ENABLED'] = server.MQTT_TLS_ENABLED
+    topic = server.MQTT_TOPIC
+
+    mqtt_client = Mqtt(application)
+
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.INFO)
+
+    @mqtt_client.on_connect()
+    def handle_connect(client, userdata, flags, rc):
+        if rc == 0:
+            logger.info('MQTT Connected successfully')
+            mqtt_client.subscribe(topic) # subscribe topic
+        else:
+            logger.info('Bad connection. Code:', rc)
+
+    @mqtt_client.on_message()
+    def handle_mqtt_message(client, userdata, message):
+        data = dict(
+                topic=message.topic,
+                payload=message.payload.decode()
+            )
+        logger.debug('Received message on topic: {topic} with payload: {payload}'.format(**data))
+        try:
+            payload = json.loads(message.payload)
+        except ValueError:
+            payload = None
+
+        if payload:
+            url = f'http://{server.DEFAULT_HOST}:{server.DEFAULT_PORT}/v2/notify'
+            headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
+            r = requests.post(url, data=json.dumps(payload), headers=headers)
```

Comment:
- can we please move this code to another module: `server.mqtt_client.py`?
- also if you could please use a class and turn these functions into methods
- then have a top-level function `run_if_enabled` you export from `server.mqtt_client.py` and call that function here